src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
+    }
+
+    $('.order-override.cancel.button').click(function() {
+      var dateToRemove = $('.ui.order-override.modal').attr('data-order-date');
```

Comment:
If we use an HTML data attribute `data-something="value"`, could be better to use the `jQuery` `data` method to get the HTML attribute content instead of `attr`. For example:

`$('.ui.order-override.modal').data('orderDate');`